addons/safestart/functions/pfh_start.sqf

```diff
@@ -4,7 +4,23 @@
 _this setVariable ["timer", _this getVariable ["params", -1]];
 
 player allowDamage false;
-ace_advanced_throwing_enabled = false;
+
+if (isClass (configFile >> "CfgPatches" >> "ace_advanced_throwing")) then {
+    ace_advanced_throwing_enabled = false;
+
+    // Edge case
+    if (isNil QGVAR(ace_throwing_setting_eh)) then {
+        GVAR(ace_throwing_setting_eh) = ["CBA_SettingChanged", {
+            params ["_setting", "_value"];
+            if (_setting == "ace_advanced_throwing_enabled") then {
+                GVAR(ace_throwing) = _value;
+                if ([] call FUNC(isActive)) then {
+                    ace_advanced_throwing_enabled = false;
+                };
+            };
+        }] call CBA_fnc_addEventHandler;
```

Comment:
If start is called twice this EH will be duplicated.  I believe this should be registered in postInit/preInit. Then this method just should do:
```
ace_advanced_throwing_enabled = false;
```

Reply:
nevermind I mis read it with the isnl, as start could happen after the fact :D - However if the Setting has already been initialised by the time start is called `GVAR(ace_throwing)` will not be set (i.e. admin trigger safe start), so setting it in start as well as the event handler is probably worth it.